src/packagers/JSPackager.js

```diff
@@ -225,12 +225,11 @@ class JSPackager extends Packager {
       // Add source map url if a map bundle exists
       let mapBundle = this.bundle.siblingBundlesMap.get('map');
       if (mapBundle) {
-        await this.write(
```

Comment:
Discovered a potential bug with template strings in Babel 7. But I think it has already been reported. Feel free to check if anyone is up for it :)

It might have been an issue in the babel config I wrote, not sure.